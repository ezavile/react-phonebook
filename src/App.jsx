import React from 'react';
import CoverImage from './components/CoverImage';

const App = () => (
  <CoverImage
    image="https://images.unsplash.com/photo-1520880867055-1e30d1cb001c?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=649616c0b471231d6b8e637e7dce500d&auto=format&fit=crop&w=1950&q=80"
    title="React Phonebook!"
  />
);

export default App;
