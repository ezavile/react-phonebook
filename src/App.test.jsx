import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

it('renders without crashing', () => {
  const appComponent = shallow(<App />);
  expect(appComponent.getElements()).toMatchSnapshot();
});
