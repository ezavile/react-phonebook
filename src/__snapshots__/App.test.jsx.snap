// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <CoverImage
    image="https://images.unsplash.com/photo-1520880867055-1e30d1cb001c?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=649616c0b471231d6b8e637e7dce500d&auto=format&fit=crop&w=1950&q=80"
    title="React Phonebook!"
/>,
]
`;
